search-app/src/cmr/search/api/association.clj

```diff
@@ -84,6 +96,30 @@
   (validate-association-content-type headers)
   (info (format "Dissociating %s [%s] from collections: %s by client: %s."
                 (name concept-type) concept-id body (:client-id context)))
-  (let [results (assoc-service/dissociate-from-collections context concept-type concept-id body)
-        status-code (association-results->status-code concept-type results)]
-    (api-response status-code results)))
+  (if-not (valid-association-args-count? concept-type
```

Comment:
nit: use `if` rather than `if-not`.

Reply:
of course flip the statements within the block after you switch. It is just more readable to me that way.